gocd/staging.bot.gocd.yaml

```diff
@@ -0,0 +1,146 @@
+      repos:
+        git: git://botmaster.suse.de/opensuse-repos.git
+        auto_update: true
+        whitelist:
+          - openSUSE:Factory:Staging:adi*.yaml
+        destination: repos
+    stages:
+    - Run:
+        jobs:
+          Run:
+            resources:
+            - repo-checker
+            tasks:
+            - script: |-
+                cd scripts
+                ./staging-installcheck.py -A https://api.opensuse.org -p Factory
+    SBot.Regular.Factory:
+      group: Factory.pkglistgen
+      lock_behavior: unlockWhenFinished
+      timer:
+        spec: 0 0 * ? * *
+      environment_variables:
+        OSC_CONFIG: /home/go/config/oscrc-staging-bot
+      materials:
+        git:
+          git: https://github.com/openSUSE/openSUSE-release-tools.git
+      stages:
+      - Run:
+          approval:
+            type: manual
+          jobs:
+            Run:
+              timeout: 0
+              resources:
+              - staging-bot
+              tasks:
+              - script: osc -A https://api.opensuse.org staging -p Factory rebuild
+              - script: osc -A https://api.opensuse.org staging -p Factory list --supersede
+              - script: osc -A https://api.opensuse.org staging -p Factory adi --by-develproject
+              - script: osc -A https://api.opensuse.org staging -p Factory select --non-interactive --merge --try-strategies
+              - script: osc -A https://api.opensuse.org staging -p Factory unselect --cleanup
+              - script: osc -A https://api.opensuse.org staging -p Factory repair --cleanup
+  
+  AdiChecker.Leap.15.1:
+    group: openSUSE.Checkers
+    lock_behavior: unlockWhenFinished
+    environment_variables:
+      OSC_CONFIG: /home/go/config/oscrc-repo-checker
+    materials:
+      script:
+        git: https://github.com/openSUSE/openSUSE-release-tools.git
+        destination: scripts
+      repos:
+        git: git://botmaster.suse.de/opensuse-repos.git
+        auto_update: true
+        whitelist:
+          - openSUSE:Leap:15.1:Staging:adi*.yaml
+        destination: repos
+    stages:
+    - Run:
+        jobs:
+          Run:
+            resources:
+            - repo-checker
+            tasks:
+            - script: |-
+                cd scripts
+                ./staging-installcheck.py -A https://api.opensuse.org -p Leap:15.1
+    SBot.Regular.Leap.15.1:
+      group: Factory.pkglistgen
+      lock_behavior: unlockWhenFinished
+      timer:
+        spec: 0 0 * ? * *
+      environment_variables:
+        OSC_CONFIG: /home/go/config/oscrc-staging-bot
+      materials:
+        git:
+          git: https://github.com/openSUSE/openSUSE-release-tools.git
+      stages:
+      - Run:
+          approval:
+            type: manual
+          jobs:
+            Run:
+              timeout: 0
+              resources:
+              - staging-bot
+              tasks:
+              - script: osc -A https://api.opensuse.org staging -p Leap:15.1 rebuild
```

Comment:
'-p' parameter won't working with simplified 'Leap:15.1' IIRC, it works with '-p Factory' though.

Reply:
yeah, that's a bug in the erb